Example.WebAPI/src/Example.Business/Models/BinaryResponse.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Example.Business.Models
+{
+    public class BinaryResponse
+    {
+        public string modifiedString { get; set; }
+        public string originalString { get; set; }
+
+        internal void ModifiedString(int v)
+        {
+            //throw new NotImplementedException();
+        }
```

Comment:
What's the purpose of this empty function?

Reply:
I am not sure that I added that to the controller, the only thing I added were the copies of what you had for cap

Reply:
@sreevescarson Did you add this function? I'm just curious about what it's purpose is